index.js

```diff
@@ -78,6 +79,16 @@ function fastifyWebsocket (fastify, opts, next) {
     oldClose.call(this, cb)
   }
 
+  function wsHandle (handle, req, res) {
+    req[kWs].resume()
```

Comment:
instead of doing this, can you use the `'addListener'` event and resume _only_ if a new `'message'` event handler is added to the websocket?

Reply:
Done :)